packages/view/src/components/ThemeSelector/ThemeSelector.tsx

```diff
@@ -1,10 +1,9 @@
 import "./ThemeSelector.scss";
-import { container } from "tsyringe";
 import { useCallback, useEffect, useState } from "react";
 
 import { debounce } from "utils";
-import type IDEPort from "ide/IDEPort";
 
+import { setPrimaryColor } from "../../services";
```

Comment:
alias 추가해서 absolute path 사용하면 어떨까요? 하지만 아직 사용하는 곳이 적으니 이후에 적용해도 좋을 것 같습니다 ㅎㅎ
```suggestion
import { setPrimaryColor } from "services";
```

Reply:
앗. @rlaqltmxm 님, 그러면 간단하게 저것들 (혹은 src 폴더 젤 상위 폴더들?)에 대한 alias 추가해주실 수 있으실까요?
(제가 직접해본적이 없어서.. ㅎㅎ)

Reply:
네 이대로 올려주시면 제가 다른 pr에서 올려보겠습니다 !